spec/PeriodicBackgroundSync-index.bs

```diff
@@ -70,28 +72,47 @@ The <code>periodicsync</code> event is considered to run <dfn>in the background<
+A [=/service worker registration=] has an associated <dfn>list of periodicsync registrations</dfn> whose element type is a [=periodicsync registration=].
 
-A <dfn>periodicsync registration</dfn> is a tuple consisting of a [=tag=], a dictionary of options, and a [=registration state=].
+A <dfn>periodicsync registration</dfn> is a tuple consisting of:
+* a [=periodicsync registration/service worker registration=] (a [=/service worker registration=]),
+* a [=periodicsync registration/tag=] (a [=string=]),
+* a dictionary of [=periodicsync registration/options=] (an [=ordered map=] whose keys are {{options/minInterval}} and values are numbers),
+* a [=periodicsync registration/time to fire=] (a number),
+* a [=count of retries=] (a number), and,
+* a [=periodicsync registration/registration state=] (an enumeration).
 
-The <dfn>tag</dfn> is a {{DOMString}}. Within one [=list of periodicsync registrations=] each [=periodicsync registration=] MUST have a unique [=tag=]. Periodic Background Sync doesn't share namespace with Background Sync, so an origin can have registrations of both types with the same tag.
+<div dfn-for="periodicsync registration">
+The <dfn>service worker registration</dfn> is the [=/service worker registration=] associated with the [=context object=] the {{PeriodicSyncManager}} belongs to.
 
-The options are a dictionary containing [=minInterval=], a long long. <dfn>minInterval</dfn> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen. [=minInterval=] is a suggestion to the user agent. The actual interval at which <code>periodicsync</code> events are fired MUST be greater than or equal to this.
-Enclosing options in a dictionary allows this spec to be extended with more options in the future without adversely affecting existing usage.
+The <dfn>tag</dfn> is a {{DOMString}}. Within one [=list of periodicsync registrations=] each [=periodicsync registration=] MUST have a unique [=periodicsync registration/tag=]. Periodic Background Sync doesn't share namespace with Background Sync, so an [=origin can have registrations of both types with the same tag.
 
-The <dfn>registration state</dfn> is one of <dfn>pending</dfn>, <dfn>waiting</dfn>, <dfn>firing</dfn>, or <dfn>reregisteredWhileFiring</dfn>. It is initially set to [=pending=].
+The <dfn>options</dfn> are a dictionary containing {{options/minInterval}}, a long long. <dfn attribute for=options>minInterval</dfn> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen. {{options/minInterval}} is a suggestion to the user agent. The actual interval at which <code>periodicsync</code> events are fired MUST be greater than or equal to this.
+Enclosing [=periodicsync registration/options=] in a dictionary allows this spec to be extended with more [=periodicsync registration/options=] in the future without adversely affecting existing usage.
```

Comment:
I wouldn't include this line

Reply:
This was a recommendation by Peter in a previous comment on this CL.